NEWS

```diff
@@ -11,6 +11,9 @@ CHANGES WITH 235:
           that case users will be prevented from correctly managing bond0
           interface using networkd.
 
+        * clock file is now moved to /var/lib/systemd/timesync/clock.
+          Distributors should change the location of the file if it is packaged.
+
```

Comment:
The %ghost directive means that the file will be removed when the package is removed/updated. But this file will be written to by systemd-timesyncd when it is stopped. Thus this is still relevant I think.